From 66e2265f8a5f85ff0b4934450fc0aaba354129c3 Mon Sep 17 00:00:00 2001
From: Antony Abee Prakash XV <antonyabee.prakashxv@adlinktech.com>
Date: Fri, 5 Jun 2020 17:20:29 +0530
Subject: [PATCH] Fix-Sony-imx219-camera-sensor-greenish-colour-issue

This patch will modify camera module-name and module-lens-name to match
the camera sensor iqfile name

Signed-off-by: Antony Abee Prakash XV <antonyabee.prakashxv@adlinktech.com>
---
 arch/arm64/boot/dts/rockchip/px30-evb-ddr3-v10-linux.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/px30-evb-ddr3-v10-linux.dts b/arch/arm64/boot/dts/rockchip/px30-evb-ddr3-v10-linux.dts
index 653e560..b0d76eb 100644
--- a/arch/arm64/boot/dts/rockchip/px30-evb-ddr3-v10-linux.dts
+++ b/arch/arm64/boot/dts/rockchip/px30-evb-ddr3-v10-linux.dts
@@ -1188,8 +1188,8 @@
                 pinctrl-0 = <&cif_clkout_m0 &cam_rst &cam_pwr>;
                 rockchip,camera-module-index = <0>;
                 rockchip,camera-module-facing = "back";
-                rockchip,camera-module-name = "TongJu";
-                rockchip,camera-module-lens-name = "CHT842-MD";
+                rockchip,camera-module-name = "default";
+                rockchip,camera-module-lens-name = "default";
                 port {
                         ucam_out: endpoint {
                                 remote-endpoint = <&mipi_in_ucam>;
-- 
1.9.1

